script/check_python_requirements.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Helper script to check that packages defined in a requirements.txt
+# file can be installed in different Python versions, it checks by
+# default the requirements.txt file for libbeat tests.
+#
+#    Usage: check_python_requirements.sh /path/to/requirements.txt
+#
+# VERSIONS environment variable can be set to a space-separated list
+# of versions of python to test with.
+#
+
+set -e
+
+BEATS_PATH=$(realpath $(dirname ${BASH_SOURCE[0]})/..)
+
+VERSIONS=${VERSIONS:-3.5 3.6 3.7 3.8 3.9-rc}
+REQUIREMENTS=${1:-${BEATS_PATH}/libbeat/tests/system/requirements.txt}
+
+if [ ! -f $REQUIREMENTS ]; then
+	echo "Requirements file doesn't exist: $REQUIREMENTS"
+	exit -1
+fi
+
+REQUIREMENTS=$(realpath $REQUIREMENTS)
```

Comment:
Will this script work out of-the-box on other environments except linux? On darwin it seems that there is no such command by default (but can be installed with `brew install coreutils`): https://stackoverflow.com/questions/3572030/bash-script-absolute-path-with-os-x

Reply:
Good point, I have added a small function that uses `cd` and `pwd` as suggested in one of the comments there.